addons/binding/org.openhab.binding.harmonyhub/src/main/java/org/openhab/binding/harmonyhub/handler/HarmonyHubHandler.java

```diff
@@ -76,6 +80,8 @@
 
     private static final int HEARTBEAT_INTERVAL = 30;
 
+    private ScheduledExecutorService buttonExecutor = Executors.newSingleThreadScheduledExecutor();
```

Comment:
@wborn that's exactly why, i should read my own commit message!   This was for #1665 ,  the desire is to queue this up in a single thread to be executed, I think my thought was the Thing executor service creates a pool of parallel thread workers which is not the desired effect. @kaikreuzer is this acceptable?